mozfun/norm/fenix_app_info/README.md

```diff
@@ -0,0 +1 @@
+@sql(../examples/fenix_app_info.sql)
```

Comment:
What is this?

Reply:
It injects the text of the example SQL file into the README. See rendered examples in https://mozilla.github.io/bigquery-etl/mozfun/bits28/

The examples are sent through dry run to verify that they work, and we also enforce formatting on them, so that we don't get stale or incorrect examples.